py/newsletter.py

```diff
@@ -10,6 +10,10 @@
 from bodynoline import _bodynoline
 from foot import _foot
 
+# USER VARIABLES
+# week number
+week = 'HAPPY WEEK 7 EVERYONE'
```

Comment:
This is fine for now, but I think eventually we will need to be putting the user strings into another file, then having this program parse through that other file, otherwise this section will get huge. @shravan1k fyi

Reply:
Good point